Generate content of the knowledge base index page (#794)
Avaiga/taipy-doc: docs/knowledge_base/demos/face_recognition.md

---
title: Real-time Face Recognition
category: demos
type: code
data-keywords: gui ai

> short-description: Explore real-time face detection and recognition demo using a Taipy UI with OpenCV.

Reviewer: ```suggestion
short-description: Real-time face detection and recognition demo using a Taipy UI with OpenCV.
```

# Understanding the Application

> This one page demo provides real-time face detection capabilities

Reviewer: ```suggestion
This demo provides real-time face detection capabilities
```

thanks to the robust [OpenCV library](https://opencv.org/).

## How to use the Application

> 1. While opening the application, you'll see yourself across your webcam. A red square

Reviewer: ```suggestion
1. While opening the application, you'll see yourself, captured by your webcam. A red square
```

   surrounding your face with someone else's name.

> 2. Train the model to recognize you by clicking the "Capture" button and giving your name

Reviewer: ```suggestion
2. Train the model to recognize you by clicking the "Capture" button and providing your name
```
